docs/DeveloperGuide.md

```diff
@@ -299,73 +290,85 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 1. User enters command to create new student entry.
-2. System displays success message and command line is cleared.
+1. System displays success message and command line is cleared.
 
    Use case ends.
 
 **Extensions**
 
 * 1a. System detects error in entered command.
     * 1a1. System displays error message and does not clear command line.
-    * 1a2. User enters new command.
-* Steps 1a1-1a2 are repeated until all details entered are correct.
-* Use case resumes from step 2.
+    * 1a2. User enters new command.<br>
+  Steps 1a1-1a2 are repeated until all details entered are correct.<br>
+  Use case resumes from step 2.<br><br>
+  
+* 1b. System detects error in parameters.
+    * 1b1. System displays error message and does not clear command line.
+    * 1b2. User enters command with correct parameters.<br>
+  Steps 1b1-1b2 are repeated until all details entered are correct.<br>
+  Use case resumes from step 2.
```

Comment:
Should we follow this standard or the standard in the use cases for pay and settle? This one has extensions for error in entered command and entered parameters as separate but in pay and settle they are considered in the same use case

Reply:
Good question